src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -12597,6 +12590,14 @@ void Unit::_ExitVehicle(Position const* exitPosition)
         SendMessageToSet(&data, false);
     }
 
+    // Default exit position to vehicle position
```

Comment:
any reason to move this code down here from where it was before ?

Reply:
I moved it because I think it makes sense to declare pos right before it's actually used, no functionality change.